assets/javascripts/discourse/components/group-tracker-first-post.hbs

```diff
@@ -0,0 +1,10 @@
+{{#if @topic.first_tracked_post}}
+  <DButton
+    class="first-tracked-post"
```

Comment:
```suggestion
    @class="first-tracked-post"
```

Reply:
I understand that non-`@` attributes can get passed directly as rendered HTML attributes - we're keeping `@class` support in DButton for backwards compatibility (for now), see https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/templates/components/d-button.hbs#L3

Reply:
Good to know, thanks for pointing this out :)